lib/msal-core/package.json

```diff
@@ -29,7 +29,7 @@
   },
   "devDependencies": {
     "@types/handlebars": "4.0.33",
-    "@types/jasmine": "^2.6.0",
+    "@types/jasmine": "2.5.41",
```

Comment:
In angular we are using 2.5.38 (committed in Rohit's PR for authority issue), will versions need to be different going forward?